addons/modules/XEH_postInit.sqf

```diff
@@ -8,8 +8,45 @@ if (isServer) then {
 
     // Public variable to track created target logics
     missionNamespace setVariable [QGVAR(targetLogics), [], true];
+
+    // Public variable to track created teleporter objects
+    missionNamespace setVariable [QGVAR(teleporters), [], true];
+
+    [QGVAR(createTeleporter), {
```

Comment:
I'm starting to think this should be extracted to a separate function. You're going to have more stuff in this file at a later point in time and it will get exponentially harder to maintain. And this is quite a long code block anyways.